lld/test/ELF/lit.local.cfg

```diff
@@ -1,4 +1,4 @@
-config.suffixes = [".test", ".s", ".ll", ".bat"]
+config.suffixes = [".test", ".s", ".ll", ".bat", ".yaml"]
```

Comment:
should use `.test` suffix and avoid adding a new suffix here

Reply:
please spend more time figuring out the convention then adding new stuff

Reply:
You've gotta be the rudest reviewer I've ever met over in two years and 95 landed PRs of working on LLVM. 

On the previous PR followed me back to my downstream project and criticized me for an honest mistake.  

Then you ask for a test for a patently correct 10 char/1 line PR - which I supplied even though it's absurd.

Now you're criticizing me for not following conventions which you clearly yourself don't have a complete grasp of - **newflash** there are literally `.yaml` [files in this same test set](https://github.com/llvm/llvm-project/blob/04c38981a9ce3e6225669c0e41cab947e3e7989f/lld/test/MachO/zippered.yaml#L4).

Please spend more time learning how to communicate with your colleagues (or I guess just have fun maintaining completely by yourself - your choice :shrug:).